pullrequest.js

```diff
@@ -36,16 +36,38 @@ function getDiffSpans(path) {
     });
 }
 
+function getIds(path) {
+    var $spans = getDiffSpans(path).closest('[id^=diff-]');
+    var $as = $spans.prev('a[name^=diff-]');
+    var ids = $as.map(function(index, a) {
```

Comment:
I think `ids` should have a `$` in front of it, since it's still a jQuery object. It'd be great if they could be changed to `$ids` in the functions below as well, since you can't treat them the exact same way you would a vanilla array (you're using `each` instead of `forEach`, for example).